### PR TITLE
Weektoweek report

### PR DESCRIPTION
**Pull request by jpcunningh**
Adds a new Week to Week report that plots glucose levels with each day in a week overlayed on the same graph.

### CODE REVIEW

**Comment by PieterGit** [comment]
@jpcunningh is this ready to test?

**Comment by jpcunningh** [comment]
It’s ready. Thanks!

File: lib/report_plugins/weektoweek.js [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+          , 'mills': sgv.mills - currDay * 24*60*60000
+          , 'noise': sgv.noise
+          , 'sgv': sgv.sgv
+          , 'type': sgv.type
+          , 'unfiltered': sgv.unfiltered
+          , 'y': sgv.y
+        });
+      });
+
+      // console.log('First sgv: ' + sgvData[0].date + ' last sgv: ' + sgvData[sgvData.length - 1].date);
```
Comment (PieterGit) [inline lib/report_plugins/weektoweek.js]: can you remove logging code and add a unit test for this report?

Comment (jpcunningh) [reply lib/report_plugins/weektoweek.js]: @PieterGit, a unit test is implemented and the logging code is removed.

**Comment by PieterGit** [comment]
This PR has been integrated with the https://github.com/nightscout/cgm-remote-monitor/tree/needs_testing 
branch. Please test that branch and report at this issue or at nightscout/cgm-remote-monitor#4169

**Comment by PieterGit** [comment]
@jpcunningh thanks for this great new report.

This PR has been tested as part of the `need_testing` branch of https://github.com/nightscout/cgm-remote-monitor/pull/4169
@blocklist_twitter @balshor @sulkaharo @jpcunningh @unsoluble @veryfancy @balshor @tynbendad have run this PR for a week. Several users have confirmed this PR to work for them.
No negative stuff popped up. Gitter link: https://gitter.im/nightscout/public?at=5c2f33d12863d8612bd78eb0

If you encounter bugs with this PR, please create a new issue and link it to this PR.
Closing PR, will be merged with https://github.com/nightscout/cgm-remote-monitor/pull/4169

**Comment by lubio** [comment]
hi

**Pull request closed without merging**